target/linux/ramips/mt7621/base-files/lib/upgrade/dna.sh

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2023 Mauri Sandberg
+#
+
+# The vendor ubi is split in ubifs volumes 0-3 and the vendor root
```

Comment:
s/ubifs volumes/UBI volumes/

UBIFS is on top of UBI and volume env1 and env2 are not UBIFS, rootfs_1 and rootfs_2 are UBI volumes with UBIFS.